contacts_vs_distance.ipynb

```diff
@@ -44,7 +44,7 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.            clr=clr_downsampled,
</pre><p>idea here to calculate on the raw data instead of the balanced data. </p><p> </p><p>to get the smoothed P(s) on the raw data, however, would require a PR to update the function that generates the smoothed cvd. It looks like a different cols dictionary could be passed to this function to generate the counts.avg.smoothed , and then another line to append th new smoothed raw P(s) to the cvd dictionary <a href="https://github.com/open2c/cooltools/blob/ba2be42c714fffb6b62ea13e7ecfbecb5f4a4d9a/cooltools/api/expected.py#L1016" target="_blank">https://github.com/open2c/cooltools/blob/ba2be42c714fffb6b62ea13e7ecfbecb5f4a4d9a/cooltools/api/expected.py#L1016</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>